cmd/nerdctl/run.go

```diff
@@ -55,6 +56,12 @@ import (
 	"github.com/spf13/cobra"
 )
 
+const (
+	nerdctlInitBinary     = "nerdctl-init"
+	tiniInitBinary        = "tini"
```

Comment:
Since nerdctl-init is backed by tini, why do we still need `tiniInitBinary`?

Reply:
Yes, we will lookup tiniInitBinary in Path